projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -3,10 +3,47 @@
 }
 
 .jp-carousel-wrap * {
-	line-height:inherit; /* prevent declarations of line-height in the universal selector */
+	line-height: inherit;
+}
+
+.swiper-pagination .swiper-pagination-bullet:not( .swiper-pagination-bullet-active ) {
+	background: #fff;
+	opacity: 0.5;
+}
+
+.swiper-container {
+	height: calc( 100vh - 90px );
+}
+
+.jp-carousel-overlay .swiper-zoom-container {
+	background-size: 200%;
+	background-repeat: no-repeat;
+	background-position: center;
+	backdrop-filter: blur( 75px ) saturate( 50% );
+	-webkit-backdrop-filter: blur( 75px ) saturate( 50% );
```

Comment:
The blur is causing some serious lag in desktop browsers. Do we need them?

Reply:
I haven't had a chance to manage performance yet, so let me look at this today. Might need to change it out for a solid color or gradient.

Reply:
this approach was reverted